docs/pipelines/overview/configuration.md

```diff
@@ -1,95 +1,368 @@
-    # A valid account vending configuration includes:
-      # account-identifiers: List of account identifiers that will be created (required).
-      account-identifiers:
-        # Alphanumeric account identifiers only. On account requests, an account
-        # will be created for each specified identifier & the account name will
-        # include the identifier. e.g. "<ACCOUNT-FAMILY>-dev"
-        - dev
-        - stage
-        - prod
-      # catalog-repositories: List of repositories used by Terragrunt catalog (optional).
-      catalog-repositories:
-        # List of repositories that contain infrastructure modules that can be easily leveraged as a catalog by delegated repositories vended by the infrastructure-root repository.
-        # For more information, see https://terragrunt.gruntwork.io/docs/features/catalog/
-        - "{{ .RepoBaseUrl }}/{{ .InfraModulesRepoName }}"
-      # github-collaborators:
-      # List of GitHub teams and their permissions automatically added to delegated repositories vended by the infrastructure-root repository.
-      # Valid permissions are: pull, triage, push, maintain and admin (in addition to custom roles if any exist)
-      # See https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/managing-repository-roles/repository-roles-for-an-organization#permissions-for-each-role
-      # for more information on GitHub repository roles.
-      #   - team: 'team-name'
-      #     permission: pull
-    sandbox:
-      account-identifiers:
-        - sandbox
-      catalog-repositories:
-        - "{{ .RepoBaseUrl }}/{{ .InfraModulesRepoName }}"
+
+More on configuration components like [aws_accounts](#aws-accounts-blocks) and [authentication_profile](#authentication-profile-blocks) can be found below.
+
+*Supported Blocks:*
+
+- `filter` (Required): A filter block that determines which units the environment is applicable to. See [Filter Blocks](#filter-blocks) for more information.
+- `authentication` (Required): An authentication block that determines how Pipelines will authenticate with cloud platforms when running Terragrunt commands. See [Authentication Blocks](#authentication-blocks) for more information.
+
+:::caution
+Every unit must be uniquely identified by a single environment block. If a unit is matched by multiple environment blocks, Pipelines will throw an error.
+:::
+
+### Authentication Profile Blocks
+
+Authentication Profile blocks are configurations used by [authentication](#authentication-blocks) blocks to have commonly re-used authentication configurations codified and referenced by multiple authentication blocks.
+
+Note that this is a configuration block that is qualified by the first label used on it. In the example below, the first label is `aws_oidc`, so the block is an `aws_oidc` authentication profile block.
+
+At this time, the only supported label that can be set for the first label is `aws_oidc`.
```

Comment:
There's that word label again - is this used with the same meaning as previously?